pyigm/clustering/xcorr_utils.py

```diff
@@ -80,7 +95,8 @@ def cross_pairs_rt(x1, y1, z1, x2, y2, z2, rbinedges, tbinedges,wrapped=True):
         y2=auxy
         z2=auxz
 
-    for i in range(len(x1) - 1):
+    # Ugly for loop
+    for i in range(len(x1)):# - 1):  JXP Bug fix?!
```

Comment:
agreed with the bug here. The loop should iterate the full extent of x1.

Reply:
removed my comment